aditional.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+type unexportedType string
+
+func ExportedFunc() unexportedType {
+	return unexportedType("some string")
```

Comment:
```suggestion
type ExportedType string

func ExportedFunc() ExportedType {
	return ExportedType("some string")
}
```
The type `unexportedType` and function `ExportedFunc` should be exported by capitalizing the first letter.  Unexported types and functions are not accessible from outside the package.